src/CqlBuilderPanel/codesSection/codesSubSection/codeSubSection/ResultsSection.tsx

```diff
@@ -76,7 +80,9 @@ export default function ResultsSection({
       },
       {
         header: "System Version",
-        accessorKey: "svsVersion",
+        accessorKey: measureModel?.includes("QDM")
```

Comment:
I think we had an idealistic view of trying to avoid model checks within components...not sure if this can be abstracted out in some way to maybe have two different model specific headers and renderers?